exonum/src/api/public/system.rs

```diff
@@ -25,18 +25,40 @@ struct MemPoolInfo {
     pub size: usize,
 }
 
-#[doc(hidden)]
+/// Contains amount of connected peers
```

Comment:
1. Please add dots here and below.
2. If I don't miss anything, we use verbs for methods and nouns for structures in documentation (examples could be found in [node/mod.rs](https://github.com/exonum/exonum/blob/master/exonum/src/node/mod.rs#L67).